### PR TITLE
Feat-HomePage

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the home page, navbar topbar and categories section.

https://user-images.githubusercontent.com/75560307/161050291-5154e542-e575-4c11-9a26-44d7415b799e.mp4

### CODE REVIEW

File: src/component/Categories/CategoriesSection.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {useAxios} from "../../utils/useAxios";
+function CategoriesSection() {
+    const  { apiResponse , loading }  = useAxios("/api/categories");
+    const { categories } = apiResponse;
+    console.log(categories);
```
Comment (coldpigli) [inline src/component/Categories/CategoriesSection.js]: remove logs when done

File: src/component/Navbar/Navbar.js [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
+  return (
+    <div className="navbar maple-flex">
+      <Link to="/">
+        <div className="nav-pill maple-flex">
+          <FaHome />
+          <h1 className="txt-m">Home</h1>
         </div>
-    )
+      </Link>
+      <Link to="/videolist">
```
Comment (coldpigli) [inline src/component/Navbar/Navbar.js]: you can easily extract this part as an array and map it through. Will make your code cleaner and your JSX component lighter. Also, you wont need to add link tags again and again

File: src/route/RoutePath.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 function RoutePath() {
     return (
         <Routes>
             <Route path='/' element={<Home />}/>
             <Route path='/playlist' element={<PlayListPage />}/>
             <Route path='/videolist' element={<VideoListingPage />}/>
+            <Route path='/liked' element={<Liked />}/>
+            <Route path='/watchlater' element={<WatchLater />}/>
+            <Route path='/history' element={<History />}/>
         </Routes>
```
Comment (coldpigli) [inline src/route/RoutePath.js:14]: Add a page for 404 routes

**Review by coldpigli — APPROVED** [review]
Have put in minor suggestions

File: src/App.css [hunk 4]
```diff
@@ -1,51 +1,109 @@
...
-
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
+@import url("https://maple-ui.netlify.app/CSS/base.css");
+:root{
+    --ecom-bg-dark : #d6d3d1;
+    --ecom-bg-light: rgb(238, 238, 238, 1);
+    --ecom-shadow: 0 3px 6px var( --ecom-bg-dark);
```
Comment (shubhamkumar648) [inline src/App.css:5]: you can use color variable in one former for consistenticy

File: src/component/Banner.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "../App.css";
+function Banner() {
+  return (
+    <div className="banner">
+      <img
+        className="banner-img"
+        src="https://res.cloudinary.com/krishanucloud/image/upload/v1648720174/banner-play_larrt1.jpg"
```
Comment (shubhamkumar648) [inline src/component/Banner.js:8]: you can add attributes alt in img tag

**Review by shubhamkumar648 — APPROVED** [review]
consider my suggestion in functionality all ready reviewed by pass! remaining code is good ,

File: src/component/Categories/CategoriesSection.js (hunk 1 above)
Comment (krishanu2911) [reply src/component/Categories/CategoriesSection.js]: Thank you for pointing out

File: src/component/Navbar/Navbar.js (hunk 2 above)
Comment (krishanu2911) [reply src/component/Navbar/Navbar.js]: Thanks for the suggestion Piyush extracted the Navbar items

File: src/route/RoutePath.js (hunk 3 above)
Comment (krishanu2911) [reply src/route/RoutePath.js:14]: Done created

File: src/component/Banner.js (hunk 5 above)
Comment (krishanu2911) [reply src/component/Banner.js:8]: Done Added

File: src/App.css (hunk 4 above)
Comment (krishanu2911) [reply src/App.css:5]: here I was just trying out things and forget to remove it I am not using these variables will remove it now

**Pull request merged**